### DISCUSSION TITLE
Message Latching

### DISCUSSION THREAD

**Discussion by jkuszmaul** [foxglove/community#1] [Q&A]
I feel like I saw some discussion of this somewhere, but not sure where....

In my custom log format, I have some configuration/parameter messages that are sent once at startup, and then never sent again. My logs are recorded in 10 minute rotating chunks (or on demand). Each log will contain the last message sent on every channel, including these one-shot channels. As such, the raw logs contain data that look like:

t = 0.1 sec: Configuration message
...
...
...
...
...
...
...
...
t=5000-5600 sec: All the data I really care about

I want some way to represent this in MCAP so that I can communicate it to studio so that studio can visualize this data effectively (without the scroll bar at the bottom having a range of 5600 sec when it should just have a range of 600 sec).

**Comment by james-rms** [comment]
This request seems to be broader in scope than just MCAP or studio, so moving to community discussion for more context.
